scripts/build/Platform/Windows/pipeline.json

```diff
@@ -22,30 +22,30 @@
             {
                 "parameter_name": "O3DE_AWS_PROJECT_NAME",
                 "parameter_type": "string",
-                "default_value": "",
+                "default_value": "AWSAUTO",
```

Comment:
The default_value should be empty here because we want to save the values in Jenkins job configuration so we can use different values for different Jenkins environments.
Same for all other default_value.